docs/hand_wire.md

```diff
@@ -169,34 +253,52 @@ Before beginning to solder, it helps to have your wire pre-bent (if using single
 
-When you're done with the columns, start with the rows in the same process, from top to bottom, and write them all down. Again, you can solder anywhere along the row, as long as it's after the diode - soldering before the diode (on the keyswitch side) will cause that row not to work.
+</details>
 
-As you move along, be sure that the Teensy is staying in place - recutting and soldering the wires is a pain!
+Cut wires to the length of the distance from the a point on each column/row to the controller.  You can solder anywhere along the row, as long as it's after the diode - soldering before the diode (on the keyswitch side) will cause that row not to work.
 
-## Additional guides
+Ribbon cable can be used to keep this extra tidy.  You may also want to consider routing the wires beneath the exisiting columns/rows.
+
+<img src="https://i.imgur.com/z2QlKfB.jpg" alt="Ribbon Cable" width="350"/>
+
+As you solder the wires to the controller make a note of which row/column is going to which pin on the controller as we'll use this data to setup the matrix when we create the firmware.
+
+As you move along, be sure that the controller is staying in place - recutting and soldering the wires is a pain!
 
-If you're more of a visual learner, or want some additional tips and something more to follow along, these two visual step by step guides may be helpful:
 
-- [BrownFox's step by step guide](https://deskthority.net/viewtopic.php?f=7&t=6050)
-- [Cribbit's modern hand wiring guide](https://geekhack.org/index.php?topic=87689.0)
 
 # Getting Some Basic Firmware Set Up
 
-From here, you should have a working keyboard once you program a firmware. Before we attach the Teensy permanently to the keyboard, let's quickly get some firmware loaded onto the Teensy so we can test each keyswitch.
+From here, you should have a working keyboard once you program a firmware.
+
+Simple firmware can be created easily using the [Keyboard Firmware Builder](https://kbfirmware.com/) website.  Recreate your layout using [Keyboard Layout Editor](http://www.keyboard-layout-editor.com), import it and recreate the matrix (if not already done as part of [planning the matrix](#planning-the-matrix).
 
-To start out, download [the firmware](https://github.com/qmk/qmk_firmware/) - we'll be using my (Jack's) fork of TMK called QMK/Quantum. We'll be doing a lot from the Terminal/command prompt, so get that open, along with a decent text editor like [Sublime Text](http://www.sublimetext.com/) (paid) or [Visual Studio Code](https://code.visualstudio.com) (free).
+Go through the rest of the tabs, assigning keys until you get to the last one where you can compile and download your firmware.  The .hex file can be flashed straight onto your keyboard, and the .zip of source files can be modified for advanced functionality and compiled locally using the method described in the collapsable section below.
+
+
+<details>
+
+<summary>Creating and compiling your firmware locally (old method)</summary>
```

Comment:
Not sure that "old" is the right word for this
```suggestion
<summary>Creating and compiling your firmware locally (command line method)</summary>
```

Reply:
Also, you could link to https://docs.qmk.fm/#/newbs_getting_started for additional information

Reply:
Done!